conda/core/link.py

```diff
@@ -1614,3 +1622,61 @@ def messages(prefix):
                 return m
     finally:
         rm_rf(path)
+
+
+def _get_best_prec_match(precs):
```

Comment:
We may want to change the name to not carry the underscore prefix since it's used in an other module (`conda.misc` also imported `.plan._get_best_prec_match` before).